### PR TITLE
chore: fix T20 rule on more files

### PR DESCRIPTION
**Pull request by kptkin**
Description
-----------

What does the PR do? Include a concise description of the PR contents.

Follow up on #9103 

- [x] I updated CHANGELOG.unreleased.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]
```python
print("LGTM")
```

File: tests/system_tests/test_core/test_mode_disabled_full.py [hunk 1]
```diff
@@ -24,10 +24,10 @@ def test_disabled_summary(user):
     run = wandb.init(mode="disabled")
     run.summary["cat"] = 2
     run.summary["nested"] = dict(level=3)
-    print(run.summary["cat"])
-    print(run.summary.cat)
+    _ = run.summary["cat"]
```
Comment (jacobromero) [inline tests/system_tests/test_core/test_mode_disabled_full.py:27]: any reason to store to `_`?

**Review by jacobromero — APPROVED** [review]

File: tests/system_tests/test_core/test_mode_disabled_full.py (hunk 1 above)
Comment (kptkin) [reply tests/system_tests/test_core/test_mode_disabled_full.py:27]: ah just wanted to emphasize that it was a intentional access to the summary attribute

Comment (jacobromero) [reply tests/system_tests/test_core/test_mode_disabled_full.py:27]: Hmm maybe a more intentional assertion here might be?
```
assert "cat" in run.summary
assert 2 == run.summary.cat
```

similarly I think below is clearer on our intention for testing
```
assert "dog" not in run.summary
```
but for the dog assertion checking the error is raised seems fine as well to me

Comment (kptkin) [reply tests/system_tests/test_core/test_mode_disabled_full.py:27]: i don't know if we change summary in disabled mode, let me verify

Comment (kptkin) [reply tests/system_tests/test_core/test_mode_disabled_full.py:27]: ah we do it works, so i can update it......

**Pull request merged**